test_module_mutual_use.chpl:26: In module 'M2':
test_module_mutual_use.chpl:29: error: use of 'a' before encountering its definition, type unknown
